app/models/accession.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: accessions
+#
+#  id               :integer          not null, primary key
+#  accession_number :string
+#  created_at       :datetime         not null
+#  updated_at       :datetime         not null
+#  degree_period_id :integer          not null
+#
+class Accession < ApplicationRecord
+  belongs_to :degree_period
+
+  validates :accession_number, presence: true,
+                               format: {
+                                 with: /\A(19|20)\d{2}_\d{3}\z/i,
+                                 message: 'must match the format `YYYY_seq`, where `YYYY` is a year between 1900 and ' \
```

Comment:
We should double check with Joe that this validation is correct. From my understanding it is, but it would be good to specifically ask him to confirm that `2023_1000` or `2023_0001` are actually impossible or just not their current practice. We can validate on whatever they want, but I curious if an accession number of `whatever` is technically valid in Archivematica so we might want to note someplace that our validations are based on current practices and not technical requirements (I'm not sure where we'd even note that)

Reply:
Joe says that [the number will always follow that format](https://mitlibraries.atlassian.net/browse/ETD-589?focusedCommentId=109536). He didn't indicate whether another format was technically possible; I'm not sure my question was clear enough to get to that.

From my perspective, though, I don't think that ETD should concern itself with what is possible in Archivematica, only what stakeholders' current needs/preferences are. Could you say more about why it's important to document that? (Fwiw, if we do, I'd put it in the readme.)

Reply:
Besides the Jira discussion, I'd say that renaming the model to `ArchivematicaAccession` and including the context you provided will help diffuse any future confusion. ("Why did we make this number like this? Oh right, it's for Archivematica.")

My hope is that, even if we can't find the ticket, stakeholders would be able to help us understand why things were the way they were and/or why they have changed. As a former digital preservationist, I know that painstaking documentation is 90% of the job. :)